### PR TITLE
[docs] Explain how to do docs PR in browser

### PR DESCRIPTION
**Pull request by rfay**
## The Problem/Issue/Bug:

The developer docs don't seem to say clearly that you can do a docs PR (and review it) using just web browser.

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/4237"></a>

### CODE REVIEW

File: docs/content/developers/testing-docs.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 This section is designed for people who want to contribute to the DDEV documentation.
 When you are going to make any changes to the documentation it is recommended that you test them locally to see what your changes look like.
 
+## Fix Docs Using Web Browser
+
+On any page in the docs
+
+1. Click the pencil in the upper right. That will take you to the the right page on GitHub. 
+2. Click the pencil button on GitHub and follow the instructions to create your change. 
+3. Save your changes and follow the prompts to create a PR.
```
Comment (rfay) [inline docs/content/developers/testing-docs.md:12]:
```suggestion
1. Click the pencil in the upper right. That will take you to the the right page on GitHub.
2. Click the pencil button on GitHub and follow the instructions to create your change.
3. Save your changes and follow the prompts to create a PR.
```

**Comment by Ellarddekoeijer** [comment]
@rfay This is great, is it possible to view the changes before creating a PR for them this way? If no maybe we should mention this in the docs.

Alternatively I would word the second paragraph differently "To start making changes you'll need a local copy of the DDEV documentation". It is no longer NEEDED to do this. Maybe say something a long the lines of "To be able to make and review changes locally....."

_I do not know how to quote / request changes on a PR, could you explain how to do that :D?_

**Comment by tyler36** [comment]
@Ellarddekoeijer 

> I do not know how to quote / request changes on a PR, could you explain how to do that :D?

If your on a open PR, look at at the changed files and click the `+` to open a WYSIWYG editor.
Use the button marked below to make a suggestion, or just make a comment.

192920089-aa70a81b-ed8b-4421-99c7-04dc2f6e82f4


See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

**Comment by Ellarddekoeijer** [comment]
Ahh right, I don't think I can suggest changes on this PR though. At least I am not seeing the option in the WYSIWYG editor

**Comment by tyler36** [comment]
I don't see it either for this one; perhaps because it has been "merged"?

**Comment by Ellarddekoeijer** [comment]
Yeah you are probably right! I'll just wait for @rfay to answer 
>  is it possible to view the changes before creating a PR for them this way?

Its easy to then update the docs with a new PR instead

**Comment by rfay** [comment]
Yes, this was merged. 

Please try a new PR, using this technique. You can see the full docs set after you create the PR, as explained here.  You can also "View file" on the files changed by the PR to see more-or-less what it will look like.

**Pull request merged**